code/modules/mob/living/carbon/human/init_signals.dm

```diff
@@ -0,0 +1,11 @@
+/mob/living/carbon/human/register_init_signals()
```

Comment:
hold on is this just componentinitialize again. why are we doing this and not just doing this inside initialize

Reply:
nvm it makes a cleaner file in living thats fine then